### ISSUE TITLE
NaN loss when ft on MSR_VTT 

### ISSUE THREAD

**Issue by mboboGO** [ArrowLuo/CLIP4Clip#59]
When I directly finetune CLIP4CLIP on msrvtt, I get NaN loss after about 100 iters.  
After checking the log, the reason may be that some missed videos in msr-vtt produce many zero input tensors. 
Thus, I change the msrvtt-dataloader to skip those missed videos by:
![image](https://user-images.githubusercontent.com/37262007/153825236-978a5aa2-cae0-4c69-8cc6-32a30a16403a.png)
, and the training become correct.

**Comment by ArrowLuo** [comment]
Hi @mboboGO, Thanks for your awesome sharing. It is a good approach to filter videos in the training phase.

**Comment by mboboGO** [comment]
Hi, I further find some videos exists but cannot be well-read, which should be also removed.
After skipping all missed and damaged videos, the ft (just on epoch) results on msr-vtt become 
![image](https://user-images.githubusercontent.com/37262007/153983522-2cdb7c63-4b7d-48c4-826e-5ab492cf11b9.png)
, which looks good.

**Comment by liuyongjie985** [comment]
Good idea to remove error video

**Comment by ForawardStar** [comment]
> Hi, I further find some videos exists but cannot be well-read, which should be also removed. After skipping all missed and damaged videos, the ft (just on epoch) results on msr-vtt become ![image](https://user-images.githubusercontent.com/37262007/153983522-2cdb7c63-4b7d-48c4-826e-5ab492cf11b9.png) , which looks good.
Hello, Very good idea. Here I wonder is your result obtained by 'meanP' or 'seqTransf' ?
